code/__HELPERS/icons.dm

```diff
@@ -1399,18 +1399,18 @@ GLOBAL_LIST_EMPTY(transformation_animation_objects)
 		animate(alpha=0)
 
 	GLOB.transformation_animation_objects[src] = transformation_objects
-	for(var/A in transformation_objects)
-		vis_contents += A
-	addtimer(CALLBACK(src, PROC_REF(_reset_transformation_animation),filter_index),time)
+	for(var/transformation_object in transformation_objects)
```

Comment:
`transforming_object`?

Reply:
Ditto here, calling vars what they are is fine and having the pattern of the iterator variable just being the list var name minus an s I would honestly argue is the best choice here as it clearly indicates that the var is one entry from the list.

Reply:
Yeah, I'm inclined to agree with ninja here, it's the pattern I decided to go for here.
I don't disagree that usually it doesn't always have to be done this way, but I think for just chipping away at single-letter variables, list name without the `s` is fine enough, and clearly shows where the variable is coming from.